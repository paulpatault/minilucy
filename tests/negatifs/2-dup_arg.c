#include <stdlib.h>

#include <printf.h>

#include <unistd.h>

int int_read() {
  int var;
  scanf("%d", &var);
  return var;
}

enum inductive_bool {
  FALSE,
  TRUE
};

int main0 (int x) {
  int y;
  
  y = x;
  
  return y;
}

int main (int argc, char* argv[]) {
  int argv_0;
  int res;
  
  while (1) {
    argv_0 = int_read();
    
    res = main0(argv_0);
    
    printf("%d ", res);
    
    fflush(0);
    
    usleep(333333);
  };
}
