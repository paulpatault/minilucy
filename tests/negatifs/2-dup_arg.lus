node f(x:int;x:int) returns (y:int);
let
  y = x;
tel
