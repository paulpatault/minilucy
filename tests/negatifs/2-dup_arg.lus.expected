[31m[1mSyntax error[0m

